protob/messages.proto

```diff
@@ -913,6 +938,274 @@ message CosiSignature {
 	optional bytes signature = 1;		// Signature
 }
 
+//////////////////////
+// Stellar messages //
+//////////////////////
+
+/**
+ * Request: Public key at the specified index
+ * @next StellarPublicKey
+ */
+message StellarGetPublicKey {
+	optional uint32 index = 1;		// Account index (derived as m/44'/148'/index')
```

Comment:
I understand why you want to enforce the BIP-32 path, but it should use `address_n` to be consistent with all other supported coins.

Reply:
> it should use `address_n`...

Will do, are you OK with me using the address_n convention but only having one element in the array? Or would you still like it to be an n-element array and have the client be in charge of passing in the `44'/144'` parts as well as the index?

I'll work on the rest of the changes as well!

Reply:
We never hardcode paths in the device as we try to avoid application specific code in the device codebase. We might revisit that in the future, but please always include full path.